authorization/invitation/service/invitation_service.go

```diff
@@ -391,25 +396,39 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
+			}
+		}
+
+		resource, e := s.Repositories().ResourceRepository().Load(ctx, inviteToIdentity.IdentityResourceID.String)
+		if e != nil {
```

Comment:
here you get `e` but you return `err`, which is wrong. Just use `err` everywhere ;)